README.md

```diff
@@ -55,14 +55,14 @@ bcl create vault put
 
 ## Docker
 
-You can create alias in your `.zshrc` or `.bashrc` and run from a Docker container
+Get latest version on TCCtl run command
 
 ```bash
-alias tcctl="docker run --rm -it -v $HOME:/root tenantcloud/tcctl"
+docker pull tenantcloud/tcctl
 ```
 
-Get latest version on TCCtl run command
+You can create alias in your `.zshrc` or `.bashrc` and run from a Docker container
 
 ```bash
-docker pull tenantcloud/tcctl
+alias tcctl="docker run --rm -it -v $HOME:/root -v $PWD:/app -w /app tenantcloud/tcctl"
```

Comment:
We need mount all $HOME dir inside container?

Reply:
yes. For use `~/.aws`, `~/.ssh`, `~/.vault-configuretion` and other folders